front_end/.eslintrc

```diff
@@ -23,7 +23,8 @@
     "no-duplicate-case": 0,
     "brace-style": 0,
     "func-names": 0,
-    "react/jsx-boolean-value": 0
+    "react/jsx-boolean-value": 0,
+    "no-console": 0
```

Comment:
Instead of turning off this rule (in general I'm finding it helpful), what do you think about just adding `//eslint-disable-line no-console` to the area you need to do a console log?

In terms of not having to do that every single time... maybe it would be worth abstracting your fetch stuff into a single function you can use in all your actions.

Feel free to file as a follow-up if you need to move on to other things